docs/subnets/create-a-fuji-subnet.md

```diff
@@ -51,219 +56,613 @@ The response should look something like:
+
+```bash
+se the arrow keys to navigate: ↓ ↑ → ←
+? What stake weight would you like to assign to the validator?:
+    Default (20)
+  ▸ Custom
+```
+
+```bash
+✔ What stake weight would you like to assign to the validator?: 30
+```
+
+We then need to specify when the validator will start validating. The time must be in the future. Custom will require to enter a specific date in 'YYYY-MM-DD HH:MM:SS' format. Let's take the default this time:
+
+```bash
+Use the arrow keys to navigate: ↓ ↑ → ←
+? Start time:
+  ▸ Start in one minute
+    Custom
+```
+
+Finally, specify how long it will be validating:
+
+```bash
+✔ Start in one minute
+Use the arrow keys to navigate: ↓ ↑ → ←
+? How long should your validator validate for?:
+  ▸ Until primary network validator expires
+    Custom
+```
+
+If we choose `Custom` here, we'll have to enter a **duration**, which is a time span expressed in hours. For example, we could say 200 days = 24 \* 200 = 4800h
+
+```bash
+✔ How long should this validator be validating? Enter a duration, e.g. 8760h: 4800h
+```
+
+The user is shown an actual date of when that is now:
+
+```bash
+? Your validator will finish staking by 2023-02-13 12:26:55:
+  ▸ Yes
+    No
+```
+
+Confirm if correct. At this point the prompt series is complete and the transaction is attempted:
+
+```bash
+NodeID: NodeID-BFa1padLXBj7VHa2JYvYGzcTBPQGjPhUy
+Network: Fuji
+Start time: 2022-07-28 12:26:55
+End time: 2023-02-13 12:26:55
+Weight: 30
+Inputs complete, issuing transaction to add the provided validator information...
+```
+
+This might take a couple of seconds, and if successful, it will print:
+
+```bash
+Transaction successful, transaction ID :EhZh8PvQyqA9xggxn6EsdemXMnWKyy839NzEJ5DHExTBiXbjV
+```
+
+This means the node has now been added as validator to the given subnet on fuji!
+
+## Join a subnet
+
+You might already have a running validator which you want to add to a specific subnet. For this we run the `join` command.
+This is a bit of a special command. The `join` command will either just _print the required instructions_ for your already running node or will attempt at configuring a config file the user provides.
+
+First also a bit of "marketing" (announcing not yet available mainnet support):
 
-## Restart Node
+```bash
+avalanche subnet join testsubnet
+Use the arrow keys to navigate: ↓ ↑ → ←
+? Choose a network to validate on (this command only supports public networks):
+  ▸ Fuji
+    Mainnet
+```
+
+In the [deploy a subnet](#deploy-a-subnet) section, we saw that a subnet is permissioned via a set of keys. Therefore not any node can be added as validator to the subnet. A holder of a control key _must_ call [addSubnetValidator](../apis/avalanchego/apis/p-chain.md#platformaddsubnetvalidator) first in order to allow the node to validate the subnet. So the tool allows the user now to verify if the node has already been permissioned ("whitelisted") to be a validator for this subnet (by calling an API in the background).
```

Comment:
subnet addValidator

Reply:
Another confusing thing is that if your start time has not begun, it will say no, even if it is scheduled

Reply:
This is most probably because we are not querying the pending validators. This should be added to a next release. https://github.com/ava-labs/avalanche-cli/issues/160

Reply:
https://github.com/ava-labs/avalanche-cli/pull/161